packages/client/tests/functional/extensions/model.ts

```diff
@@ -874,5 +874,9 @@ testMatrix.setupTestSuite(
   },
   {
     skipDefaultClientInstance: true,
+    skipEngine: {
+      from: ['wasm'],
+      reason: 'ytest',
```

Comment:
?

Reply:
This can be removed, it no longer fails with `unwrap_throw` on Wasm, most tests pass, but there are still errors like

```
    expect(jest.fn()).toHaveBeenCalledTimes(expected)

    Expected number of calls: 4
    Received number of calls: 0
```

due to logs not being emitted in Wasm. This is a correctness issue that doesn't block the vertical slice.